cs/cli/vw_clr.vcxproj

```diff
@@ -36,43 +28,14 @@
     <RunCodeAnalysis>true</RunCodeAnalysis>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
```

Comment:
>)|$(Platform) [](start = 44, length = 13)

Are we removing $Platform because x64 and win32 have identical settings?

Reply:
Yes. Being more compact makes it easier to read, and vcxproj files can be incredibly difficult to understand even when reasonably clean.

Reply:
Got it.  Thanks